File: 1480.Running-Sum-of-1d-Array/1480.cpp
class Solution {
public:
    vector<int> runningSum(vector<int>& nums) {
        int l = nums.size();
        vector<int> sum;
        int addSum=0;
        for (int i=0;i<l;++i){
                addSum += nums[i];
                sum.push_back(addSum);
        }
        return sum;
    }
};

File: 1480.Running-Sum-of-1d-Array/1480.rs
impl Solution {
    pub fn running_sum(nums: Vec<i32>) -> Vec<i32> {
        let mut sum: Vec<i32> = Vec::new();
        let mut addSum: i32 = 0;
        for num in &nums {
            addSum+=num;
            sum.push(addSum);
    }
        return sum;
    }
}
